feature-group-definitions/offscreen-canvas.yml

```diff
@@ -1,14 +1,13 @@
 spec: https://html.spec.whatwg.org/multipage/canvas.html#the-offscreencanvas-interface
 caniuse: offscreencanvas
 usage_stats: https://chromestatus.com/metrics/feature/timeline/popularity/1624
-# status:
-#   is_baseline: false
-#   # since: future
-#   support:
-#     chrome: "80"
-#     edge: "80"
-#     firefox: "105"
-#     safari: "16.4"
+status:
+  baseline: false
```

Comment:
I've filed https://bugs.chromium.org/p/chromium/issues/detail?id=1507769 about the `commit()` method. I think this is a case where we'll need a note, but it doesn't make sense to discourage use of the feature as a whole.

Reply:
`commit()` is actually in the process of being removed from the standards https://github.com/whatwg/html/pull/3872 and is effectively replaced by `requestAnimationFrame`.  
Quite unfortunately that PR is stalled for a while. I'll see if I get some free time to get it moving again if possible.
Also to be noted, it was supposed to affect *all* canvas rendering context types, not just 2D, but nobody implemented it for anything else.
So overall, discouraging its use may not be a bad thing.

Reply:
For offscreen-canvas, d9de343 drops `commit()` and dca2ddc updates the status.